global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "3.1.100"
+    "version": "3.1.200"
   }
 }
```

Comment:
I had to change this to `3.1.200` after an update to VS 2019 which included a new minor revision of NET Core SDK. Otherwise `dotnet` commands would not work...

Questions for the team and @Fosol specifically: 
- Is there any reason to keep this file in source control? 
- Can it be removed safely?

I read somewhere that if not present `dotnet` will use "latest" by default but wanted to double check.

Reply:
The file is not "required" according to blogs, but it was actually required to fix bugs in vscode.  Without it omni-sharp is unable to parse C# code correctly.

Reply:
Okay good to know. Can you confirm it still needed? All I did was update my local VS 2019 and after that I didn't have 3.1.100 anymore so it must be a drop in replacement.

I don't believe we need to update all backend projects to use this specific version. They should still be fine as this is still 3.1 not 3.2